iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -270,6 +270,7 @@ class OrgUnitChangeRequestWriteSerializer(serializers.ModelSerializer):
     class Meta:
         model = OrgUnitChangeRequest
         fields = [
+            "id",
```

Comment:
had to add it as It's needed for update change request status: 

https://github.com/BLSQ/iaso/pull/1311/files#diff-e3e9b496d0cbe3812bd9f7750ff0d0a93277e5babaa07d22149492c7f530e373R75

Reply:
I don't think that's necessary because you are performing a `patch` [at the mentioned line](https://github.com/BLSQ/iaso/pull/1311/files#diff-e3e9b496d0cbe3812bd9f7750ff0d0a93277e5babaa07d22149492c7f530e373R75):

```python
 iaso_client.patch(f"/api/orgunits/changes/{proposal_review['id']}/", json=validation)
```

And `patch` won't trigger this serializer, [but `OrgUnitChangeRequestReviewSerializer` instead](https://github.com/BLSQ/iaso/blob/e7bc7fd17d4a1d93d05354c05859e7f5ad8d60d2/iaso/api/org_unit_change_requests/views.py#L62-L63).

Reply:
Finally it should stay, when I remove it, on my local env, It's ending up something like :

`Traceback (most recent call last):
  File "/home/fleury/applications/python/iaso/setuper/setuper.py", line 74, in <module>
    setup_review_change_proposal(account_name, iaso_client=iaso_client)
  File "/home/fleury/applications/python/iaso/setuper/review_change_proposal.py", line 75, in setup_review_change_proposal
    iaso_client.patch(f"/api/orgunits/changes/{proposal_review['id']}/", json=validation)
KeyError: 'id'`

Reply:
OK I understand now, you need to get back the ID from the POST result payload.

So let's keep it, but maybe add `read_only=True` to it. Also some unit tests will need to be fixed.

Reply:
> So let's keep it, but maybe add `read_only=True` to it. Also some unit tests will need to be fixed.

Added `read_only=True` . It seems, no broken tests!